clients/drcachesim/tracer/syscall_pt_trace.cpp

```diff
@@ -91,18 +94,20 @@ syscall_pt_trace_t::init(void *drcontext, char *pt_dir_name,
         return false;
     }
     drcontext_ = drcontext;
-    open_file_func_ = open_file_func;
+    open_file_ex_func_ = open_file_ex_func;
     write_file_func_ = write_file_func;
     close_file_func_ = close_file_func;
     pttracer_handle_ = { drcontext, nullptr };
     pttracer_output_buffer_ = { drcontext_, nullptr };
     std::string output_file_name(pt_dir_name);
     output_file_name +=
         "/" + std::to_string(dr_get_thread_id(drcontext_)) + "." + OUTFILE_SUFFIX_PT;
-    output_file_ = open_file_func_(output_file_name.c_str(), DR_FILE_WRITE_REQUIRE_NEW);
+    // TODO i#5505: Pass the window-id to support windowed traces.
+    output_file_ = open_file_ex_func_(output_file_name.c_str(), DR_FILE_WRITE_REQUIRE_NEW,
+                                      dr_get_thread_id(drcontext_), 0);
 
-    /* Create a buffer to store the data generated by drpttracer. For syscall traces, only
-     * the PT data is dumped, and the sideband data is not included.
+    /* Create a buffer to store the data generated by drpttracer. For syscall
+     * traces, only the PT data is dumped, and the sideband data is not included.
```

Comment:
Not seeing any change here? Accidental reformat?

Reply:
Not sure. I assumed it was clang-format's doing. Reverted.